pkg/tsdb/loki/framer.go

```diff
@@ -0,0 +1,52 @@
+package loki
+
+import (
+	"encoding/json"
+	"strconv"
+	"time"
+
+	"github.com/grafana/grafana-plugin-sdk-go/data"
+)
+
+type lokiResponse struct {
+	Streams []lokiStream `json:"streams"`
+}
+
+type lokiStream struct {
+	Stream data.Labels `json:"stream"`
+	Values [][2]string `json:"values"`
+}
+
+func lokiBytesToLabeledFrame(msg []byte) (*data.Frame, error) {
+	rsp := &lokiResponse{}
+	err := json.Unmarshal(msg, rsp)
```

Comment:
@gabor -- although a byte>frame process exists, note that the structure is slightly different (one line for each label set!) so great way to reuse that.  

This is a good place to explore how we can use labeled frames rather than a new frame for each label set

Reply:
i see, makes sense. could you maybe rename it to something that indicates we are doing something different here, like hmm.. not sure :) `live_framer.go` ? `wide_framer.go` (though it's not really "wide" frame as described in the grafana-docs.. we need names for these dataframe-types 😄 ) ? something else?

i'm just worried that now we have `frame.go`, `framer.go`, that it would get confusing.